ui-admin/src/providers/ConfigProvider.tsx

```diff
@@ -42,7 +43,7 @@ export default function ConfigProvider({ children }: { children: React.ReactNode
     })
   }, [])
   if (error) {
-    return <p>{error}</p>
+    return <ServiceUnavailable error={error}/>
```

Comment:
I think this might be a case where we don't want the admin and participant UIs to be the same.   (and then this also saves having to wrap the admin UI in i18nProvider).